html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/Eduroam.pm

```diff
@@ -0,0 +1,95 @@
+package pfappserver::Form::Config::Authentication::Source::Eduroam;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::Eduroam
+
+=cut
+
+=head1 DESCRIPTION
+
+Form definition to create or update an Eduroam authentication source.
+
+=cut
+
+use strict;
+use warnings;
+
+use HTML::FormHandler::Moose;
+
+extends 'pfappserver::Form::Config::Authentication::Source';
+with 'pfappserver::Base::Form::Role::Help';
+
+has_field 'tlrs1_server_address' => (
+    type        => 'Text',
+    label       => 'TLRS1 server address',
+    required    => 1,
+    tags        => {
+        after_element   => \&help,
+        help            => 'Eduroam TLRS1 server address',
+    },
+);
+
+has_field 'tlrs2_server_address' => (
+    type        => 'Text',
+    label       => 'TLRS2 server address',
+    required    => 1,
+    tags        => {
+        after_element   => \&help,
+        help            => 'Eduroam TLRS2 server address',
+    },
+);
+
+has_field 'tlrs_radius_secret' => (
```

Comment:
It can be different but the provided doc will ask to set the same (easier) since we need to create new tlrs entries on eduroam for PacketFence